packages/react/package.json

```diff
@@ -111,13 +111,15 @@
     "@babel/runtime": "7.11.2",
     "@chakra-ui/react": "2.4.3",
     "@cosmos-kit/core": "^0.27.13",
+    "@cosmos-kit/icns": "^0.0.1",
```

Comment:
I think adding `@cosmos-kit/icns` inside of `@cosmos-kit/react` is sort of like adding it to the core... is there a way to move this logic in the `@cosmos-kit/icns` package?

Ideally we can load everything as a plugin w/o modifying `@cosmos-kit/core` or `@cosmos-kit/react`

Reply:
Make sense, I will move the logic!